src/background/index.js

```diff
@@ -1,22 +1,23 @@
 // @ts-check
 
-const { ipcRenderer, remote } = require('electron')
+const { ipcRenderer } = require('electron')
 const path = require('path')
 
 const logger = require('../logger')
 
 /** @typedef {import('../utils/types').BackgroundProcess} BackgroundProcess */
 /** @typedef {import('../utils/types').IpcResponse} IpcResponse */
 
-const [modulePath, argsJSON] = process.argv.slice(-2)
-const userDataPath = remote.app.getPath('userData')
+const [modulePath, argsJSON, userDataPath, mode] = process.argv.slice(-4)
 const label = path.basename(modulePath)
 
+// TODO: Used by renderer processes to determine if in development mode or not. There's probably a better way to do this
+window.mode = mode
```

Comment:
window.mode is either `"development"` or `"production"` and is defined by a value passed to `webPreferences.additionalArguments` when creating a `BrowserWindow` in the main process